spec/decorators/event_decorator_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe EventDecorator do
+  describe '#pretty_event_type' do
+    it 'converts all events types to strings' do
+      Event.event_types.keys.each do |event_type|
+        event = build_stubbed(:event, event_type: event_type)
+
+        expect(event.decorate.pretty_event_type).to be_a(String)
```

Comment:
One rule of thumb I follow when writing tests is that I have to make the test fail at least once, so that I know I'm testing the right thing.  One way to do that is to change the decorator so that it returns any string other than the expected one, such as `foo`. It should fail, but this test passes.

I don't think it's necessary to loop through all event types. What we want to verify here is that when an event is passed in to the decorator, it returns a specific string based on a localization. I would rewrite the test as follows:

```ruby
describe '#pretty_event_type' do
  it 'returns the localized text corresponding to the event type' do
    event = build_stubbed(:event, event_type: :email_changed)
    decorator = EventDecorator.new(event)

    expect(decorator.pretty_event_type).to eq t('event_types.email_changed')
  end
end
```